author: Milan P. Stanić <mps@arvanta.net>
date: 2026-09-27 17:42:50 +0000

disable get console test because it needs root access.
tested on bare metal as root that it works in that case
but don't work on alpine CI and builders

--- a/tests/libcommon.at	2026-03-18 13:49:48.359387864 +0000
+++ b/tests/libcommon.at	2026-09-27 17:25:14.512764313 +0000
@@ -16,7 +16,3 @@
 UNITTEST_MEMCHECK([$abs_builddir/libcommon/libcommon-test02], [0], [], [experr])
 AT_CLEANUP
 
-AT_SETUP([test get console])
-AT_KEYWORDS([libcommon unittest])
-UNITTEST_MEMCHECK([$abs_builddir/libcommon/libcommon-test03])
-AT_CLEANUP
